spark/src/main/java/org/apache/iceberg/spark/source/Reader.java

```diff
@@ -158,11 +129,19 @@
     } else {
       this.localityPreferred = false;
     }
-
     this.schema = table.schema();
     this.io = io;
     this.encryptionManager = encryptionManager;
     this.caseSensitive = caseSensitive;
+    // override logic to check when batched reads is enabled by turning off batched reads
+    boolean disableBatchedReads =
+            options.get("iceberg.read.disablevectorizedreads").map(Boolean::parseBoolean).orElse(false);
```

Comment:
Options that enable/disable should use "enabled". That avoids users needing to think about whether options enable or disable - true always enables and false always disables.

I recommend using `iceberg.read.parquet-vectorized.enabled`.